doc/Administration-guide/System-Configuration/General-configuration/index.md

```diff
@@ -4,4 +4,183 @@ title: General configuration
 permalink: /:path/
 sort_idx: 10
 ---
-Kaa node configuration. Inter-node communication and networking config.
+
+After [Kaa installation]({{root_url}}Administration-guide/System-installation/), configuration files for each Kaa component will be extracted into the
```

Comment:
_Page not found_ for all links. Include _variables.md_ file to capture _root_url_.